server/services/applicationService.test.ts

```diff
@@ -376,7 +376,7 @@ describe('ApplicationService', () => {
       expect(result).toEqual(paginatedResponse)
 
       expect(applicationClientFactory).toHaveBeenCalledWith(token)
-      expect(applicationClient.dashboard).toHaveBeenCalledWith(1, 'createdAt', 'asc', {})
+      expect(applicationClient.dashboard).toHaveBeenCalledWith(1, 'createdAt', 'desc', {})
```

Comment:
do you think we should go a little further and use this opportunity to document our intention here?

e.g. by describing 'what' and 'why' in our scenario e.g.

```ts
it('calls the dashboard client method with sorting options to display newest first', async () => {
  ...
}
```